src/askvm/resources/math/minus.ts

```diff
@@ -3,6 +3,12 @@ import { any, resource } from '../../lib';
 export const minus = resource({
   type: any,
   async resolver(a: number, b: number): Promise<number> {
-    return a - b;
+    if (typeof b === 'undefined') {
```

Comment:
Let's update the TypeScript argument type for `b` too

Reply:
Good catch!   
I wonder why tsc doesn't generate any warning in such cases.